deploy/crds/pravega.pravega.io_pravegaclusters_crd.yaml

```diff
@@ -253,6 +253,1093 @@ spec:
                       access. Options are "LoadBalancer" and "NodePort". By default,
                       if external access is enabled, it will use "LoadBalancer"
                     type: string
+                  ontrollerInitContainers:
```

Comment:
ontrollerInitContainers needs to be corrected

Reply:
fixed